### PR TITLE
Fixed statics bug

### PR DESCRIPTION
**Pull request by nicolinap**
Code now accounts for capitals in feature branches

### CODE REVIEW

File: actions/utils/helpers/helper.py [hunk 1]
```diff
@@ -83,6 +83,7 @@ def get_branch_name():
 
 def get_review_number():
     branch_name = get_branch_name()
+    branch_name = branch_name.lower()
```
Comment (dralquinta) [inline actions/utils/helpers/helper.py]: Here you can do something like branch_name=get_branch_name().lower()

File: actions/utils/helpers/helper.py [hunk 2]
```diff
@@ -83,6 +83,7 @@ def get_branch_name():
...
     rp_number = branch_name.split('feature/')[1]
     if '.' in rp_number:
         rp_number = rp_number.replace('.', '_')
```
Comment (dralquinta) [inline actions/utils/helpers/helper.py:88]: Does this take under account anything that comes after feature? Like weird names of the branch?

**Review by dralquinta — CHANGES_REQUESTED** [review]
See changes inline

File: actions/utils/helpers/helper.py (hunk 2 above)
Comment (nicolinap) [reply actions/utils/helpers/helper.py:88]: Yes the next couple of lines do that

**Pull request merged**